archiver.go

```diff
@@ -160,7 +160,24 @@ func selectEndpoint(endpoints []string) (string, error) {
 		return "", ErrEndpointsEmpty.New()
 	}
 
-	// TODO check which endpoint to use
+	for _, ep := range endpoints {
```

Comment:
Instead of this, we can have a slice of endpoints and do something like:
```go
var endpointsOrder = []string{"git://","https://","http://"}
[...]
for _, epo := range endpointsOrder {
    for _, ep := range endpoints {
 		if strings.HasPrefix(ep, epo) {
 			return ep, nil
 		}
 	}
}

return endpoints[0],nil
```